metricbeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -62,20 +64,62 @@ may want to change that behavior, so just edit the YAML file and modify them:
   value: changeme
 ------------------------------------------------
 
+[float]
+===== Red Hat OpenShift configuration
+
+If you are using Red Hat OpenShift, you need to specify additional settings in
+the manifest file and enable the container to run as privileged.
+
+// REVIEWERS: I wanted to make it clearer as to where this info needs to go. Is
+// this correct? 
+
+. In the manifest file, modify the `metricbeat-daemonset-modules` ConfigMap, and
+specify the following settings under `kubernetes.yml` in the `data` section:
++
+[source,yaml]
+-----
+hosts: ["https://${HOSTNAME}:10250"]
+bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+ssl.certificate_authorities:
+  - /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+-----
+
+. Under the `metricbeat` ClusterRole, add the following resources:
++
+[source,yaml]
+-----
+  - nodes/metrics
+  - nodes/stats
+-----
+
+. Grant the `metricbeat` service account access to the privileged SCC:
++
+[source,shell]
+-----
+oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:metricbeat
+-----
++
+This command enables the container to be privileged as an administrator for
+OpenShift.
+
+
 [float]
 ==== Deploy
 
-Metricbeat gets some metrics from https://github.com/kubernetes/kube-state-metrics#usage[kube-state-metrics],
-you will need to deploy it if it's not already running.
+Metricbeat gets some metrics from https://github.com/kubernetes/kube-state-metrics#usage[kube-state-metrics].
+If `kube-state-metrics` is not already running, deploy it now.
+
+// REVIEWERS: Should we show the command for deploying kube-state-metrics? Looks
+// like it's kubectl apply -f kubernetes. Is that correct?
```

Comment:
Yes, it is just this command, but they need to download the files. We might link to their deployment documentation (https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment)